scipy/optimize/tests/test_hungarian.py

```diff
@@ -41,34 +44,379 @@ def test_linear_sum_assignment():
          []),
     ]:
         cost_matrix = np.array(cost_matrix)
-        row_ind, col_ind = linear_sum_assignment(cost_matrix)
+        row_ind, col_ind = _hungarian.linear_sum_assignment(cost_matrix)
         assert_array_equal(row_ind, np.sort(row_ind))
         assert_array_equal(expected_cost, cost_matrix[row_ind, col_ind])
 
         cost_matrix = cost_matrix.T
-        row_ind, col_ind = linear_sum_assignment(cost_matrix)
+        row_ind, col_ind = _hungarian.linear_sum_assignment(cost_matrix)
         assert_array_equal(row_ind, np.sort(row_ind))
         assert_array_equal(np.sort(expected_cost),
                            np.sort(cost_matrix[row_ind, col_ind]))
 
 
 def test_linear_sum_assignment_input_validation():
-    assert_raises(ValueError, linear_sum_assignment, [1, 2, 3])
+    assert_raises(ValueError, _hungarian.linear_sum_assignment, [1, 2, 3])
 
     C = [[1, 2, 3], [4, 5, 6]]
-    assert_array_equal(linear_sum_assignment(C),
-                       linear_sum_assignment(np.asarray(C)))
-    assert_array_equal(linear_sum_assignment(C),
-                       linear_sum_assignment(np.matrix(C)))
+    assert_array_equal(_hungarian.linear_sum_assignment(C),
+                       _hungarian.linear_sum_assignment(np.asarray(C)))
+    assert_array_equal(_hungarian.linear_sum_assignment(C),
+                       _hungarian.linear_sum_assignment(np.matrix(C)))
```

Comment:
`np.matrix` usage was removed in another PR, hence this one needs a rebase. Should be a painless one; since these changes anyway need reverting.

Reply:
I have simply removed the lines involving "np.matrix".